overlord/servicestate/servicestate.go

```diff
@@ -42,14 +45,93 @@ import (
 	"github.com/snapcore/snapd/wrappers"
 )
 
+type UserScope struct {
+	Names   []string
+	Keyword bool
```

Comment:
nitpick: preference would be to have an enumeration type:

type UserSelection int

UserSelectionSelf
UserSelectionAll
UserSelectionList

perhaps?

Reply:
This was so much more elegant, thank you!